Database-Features/database-metadata-backup-concept.md

```diff
@@ -161,14 +159,14 @@ Edit config file with the following information from:
 Run restore.sh
 
 
-```"code-java"
+```shell
 ./restore.sh or bash restore.sh 
 ```
 ## Additional References
 
-* <https://www.exasol.com/support/secure/attachment/78805/metadatabackup_v20190418.tar.gz>
-* <https://community.exasol.com/t5/database-features/create-ddl-for-the-entire-database/ta-p/1417>
+* [Create DDL for the entire Database](https://exasol.my.site.com/s/article/Create-DDL-for-the-entire-Database)
 * <https://www.exasol.com/support/browse/IDEA-371>
```

Comment:
This link is no longer valid

Reply:
Thanks Eagle eye Gerardo!

I thought I leave it as is to replace later when Ideation portal is live.
Otherwise I for sure forget.